web/src/lib/domain-logic/sidebarCardsHelpers.ts

```diff
@@ -44,20 +44,17 @@ export const getVisibleCertifications = (
     return !business?.preferences.hiddenCertificationIds.includes(it.id);
   });
 };
-export const sortFundings = (fundings: Funding[], userData?: UserData): Funding[] => {
+export const sortFundingsForUser = (fundings: Funding[], userData?: UserData): Funding[] => {
   const initialSorting = fundings.sort((a, b) => {
     const nameA = a.name.toUpperCase(); // ignore upper and lowercase
     const nameB = b.name.toUpperCase();
     if (FundingStatusOrder[a.status] < FundingStatusOrder[b.status]) {
       return -1;
     } else if (FundingStatusOrder[a.status] > FundingStatusOrder[b.status]) {
       return 1;
-    } else if (nameA < nameB) {
-      return -1;
-    } else if (nameA > nameB) {
-      return 1;
     }
-    return 0;
+
+    return nameA.localeCompare(nameB);
```

Comment:
Addressing this comment from related InvestNewark PR: https://github.com/newjersey/navigator.business.nj.gov/pull/8339/files#r1700764928